src/argus_ticket_rt.py

```diff
@@ -25,15 +25,12 @@ def import_settings(cls):
             LOG.exception("Could not import settings for ticket plugin.")
             raise TicketPluginException(f"Request Tracker: {e}")
 
-        if (
-            "username" not in authentication.keys()
-            or "password" not in authentication.keys()
-        ):
+        if "token" not in authentication.keys():
```

Comment:
You no longer support username/password at all? Why?

Reply:
That was an oversight, I changed it accordingly in `create_client`, but accidentally forgot to change it here as well.